### DISCUSSION TITLE
Add Currency Symbol For Lukso

### DISCUSSION THREAD

**Discussion by Pritom-sarker** [lukso-network/LIPs#215] [General]
I would like to propose to add a currency symbol to Lukso that would replace LYX. 

Ŀ this is the symbol we should use. I initially wanted to use one of the 3 the hexagons in unicode (⬣) but they are hard to read on small screens and for near-sighted people (like me lol) so they'll often look like circles.

**Use cases would be same as ETH Ξ :**
IoT
DApps
Websites
UI/UX
etc...

Unicode U+013F 

https://charbase.com/013f-unicode-latin-capital-letter-l-with-middle-dot
https://en.wikipedia.org/wiki/Latin_Extended-A

![image_2023-06-23_22-56-22 (1)](https://github.com/lukso-network/LIPs/assets/33182615/d9109efc-b1e0-434f-baf8-bd7233d55a2a)

**Comment by 0xAzures** [comment]
Honestly, I am reading this on a iPhone 12 mini, and it is hard to read. The dot right of the L might even be recognized as dirt on the screen...just my 2cents. I have no better idea.
